test/sinks.test.js

```diff
@@ -290,7 +290,7 @@ describe('asset-pipe-sink-gcs', () => {
 
         let results = [];
 
-        for (let i = 0; i < 10; i++) {
+        for (let i = 0; i < 5; i++) {
```

Comment:
I can no longer run 5 * 10 in quick succession. The express test server hangs up. Dropping it to 5 * 5 works. Didn't check if I could do more.
I think this just means we lost some perf with the changes.

Reply:
Seems that on CI, I can't even get 5 * 5. I'm guessing all that unpacking is costing us here.